src/main/java/de/tum/in/www1/artemis/web/rest/dto/StatsForDashboardDTO.java

```diff
@@ -28,7 +30,7 @@
 
     private List<TutorLeaderboardDTO> tutorLeaderboardEntries;
 
-    public StatsForInstructorDashboardDTO() {
+    public StatsForDashboardDTO() {
```

Comment:
please add a javadoc stating that the empty constructor is needed by Jackson

Reply:
Done